src/garage/torch/algos/ddpg.py

```diff
@@ -136,11 +143,12 @@ def train_once(self, itr, paths):
         ])
 
         last_average_return = np.mean(self.episode_rewards)
-        for train_itr in range(self.n_train_steps):
-            if self.replay_buffer.n_transitions_stored >= self.min_buffer_size:  # noqa: E501
+        for _ in range(self.n_train_steps):
+            if (self.replay_buffer.n_transitions_stored >=
+                    self.min_buffer_size):
                 self.evaluate = True
                 samples = self.replay_buffer.sample(self.buffer_batch_size)
-                qf_loss, y, q, policy_loss = torch_to_np(
+                qf_loss, y, q, policy_loss = torch_to_np(  # pylint: disable=unbalanced-tuple-unpacking  # noqa: E501
```

Comment:
i think if you change the implementation of `torch_to_np`/`np_to_torch` to tuple comprehensions (rather than the naive for/append loops) then pylint might be able to infer these lengths. 

if that doesn't fix it, let me know and i'll disable the rule.

Reply:
changing to tuple comprehensions fixed the issue

Reply:
awesome!